File: micro-ui/src/const/crud/portal/archive/lvye/routineCheck.js

let dictYesNo=[
    {label: '合格', value: '1'},
    {label: '不合格', value: '0'}
]
export const  option={
    align:'center',
    indexLabel: '序号',
    border: true,
    index: true,
    editBtn:false,
    delBtn:false,
    addBtn:false,
    viewBtn:false,
    menu:true,
    menuWidth:100,
    card:true,
    column:[
        { label: "检查民警",prop: 'mj',slot:true,viewDisplay:false},

        { label: "检查时间",prop: 'checkTime',type: 'datetimerange',
            search:true,
            hide:true,
            format: 'yyyy-MM-dd HH:mm',
            valueFormat: 'yyyyMMddHHmm'
        },
        { label: "Ⅰ检查民警",prop: 'jcmjZh',hide:true},
        { label: "Ⅱ检查民警",prop: 'jcmjTwoZh',hide:true},
        { label: "入住登记制度",prop: 'rzdjzd',},
        { label: "入住登记制度说明",prop: 'rzdjzdBz',dicData:dictYesNo,hide:true},

        { label: "来访登记制度",prop: 'lfdjzd',dicData:dictYesNo},
        { label: "来访登记制度说明",prop: 'lfdjzdBz',dicData:dictYesNo,hide:true},

        { label: "值班巡查制度",prop: 'zbxczd',dicData:dictYesNo,},
        { label: "值班巡查制度说明",prop: 'zbxczdBz',dicData:dictYesNo,hide:true},

        { label: "财务报告制度",prop: 'cwbgzd',dicData:dictYesNo},
        { label: "财务报告制度说明",prop: 'cwbgzdBz',dicData:dictYesNo,hide:true},

        { label: "报告制度",prop: 'bgzd',dicData:dictYesNo},
        { label: "报告制度说明",prop: 'bgzdBz',dicData:dictYesNo,hide:true},

        { label: "信息系统使用",prop: 'xxxt',dicData:dictYesNo},
        { label: "信息系统使用说明",prop: 'xxxtBz',dicData:dictYesNo,hide:true},

        { label: "上传及时",prop: 'scjs',dicData:dictYesNo},
        { label: "上传及时说明",prop: 'scjsBz',dicData:dictYesNo,hide:true},


        { label: "消防设备",prop: 'xfsbjhyq',dicData:dictYesNo},
        { label: "消防设备说明",prop: 'xfsbjhyqBz',dicData:dictYesNo,hide:true},

        { label: "当天录入信息齐全",prop: 'dtlrxx',dicData:dictYesNo},
        { label: "当天录入信息齐全说明",prop: 'dtlrxxBz',dicData:dictYesNo,hide:true},

        { label: "上传数量和录入数量",prop: 'scslylrslxf',dicData:dictYesNo},
        { label: "上传数量和录入数量说明",prop: 'scslylrslxfBz',dicData:dictYesNo,hide:true},

        { label:"检查单位", prop:"jcdwZh",hide:true},
        { label:"检查时间", prop:"jcsj",
            format: 'yyyy-MM-dd HH:mm',
            type: 'datetime',
            valueFormat: 'yyyy-MM-dd HH:mm'
        },
    ]
}
let spanWith=12
export const    optionDetail= {
    card: true,
    group: [
        {
            label: '检查内容',
            prop: 'checkInfo',
            icon: 'el-icon-info',
            column: [
                { label: "入住登记制度",prop: 'rzdjzd',dicData:dictYesNo,span:spanWith},
                { label: "入住登记制度说明",prop: 'rzdjzdBz',dicData:dictYesNo,span:spanWith},

                { label: "来访登记制度",prop: 'lfdjzd',dicData:dictYesNo,span:spanWith},
                { label: "来访登记制度说明",prop: 'lfdjzdBz',dicData:dictYesNo,span:spanWith},

                { label: "值班巡查制度",prop: 'zbxczd',dicData:dictYesNo,span:spanWith},
                { label: "值班巡查制度说明",prop: 'zbxczdBz',dicData:dictYesNo,span:spanWith},

                { label: "财务报告制度",prop: 'cwbgzd',dicData:dictYesNo,span:spanWith},
                { label: "财务报告制度说明",prop: 'cwbgzdBz',dicData:dictYesNo,span:spanWith},

                { label: "报告制度",prop: 'bgzd',dicData:dictYesNo,span:spanWith},
                { label: "报告制度说明",prop: 'bgzdBz',dicData:dictYesNo,span:spanWith},

                { label: "信息系统使用",prop: 'xxxt',dicData:dictYesNo,span:spanWith},
                { label: "信息系统使用说明",prop: 'xxxtBz',dicData:dictYesNo,span:spanWith},

                { label: "上传及时",prop: 'scjs',dicData:dictYesNo,span:spanWith},
                { label: "上传及时说明",prop: 'scjsBz',dicData:dictYesNo,span:spanWith},


                { label: "消防设备",prop: 'xfsbjhyq',dicData:dictYesNo,span:spanWith},
                { label: "消防设备说明",prop: 'xfsbjhyqBz',dicData:dictYesNo,span:spanWith},

                { label: "当天录入信息齐全",prop: 'dtlrxx',dicData:dictYesNo,span:spanWith},
                { label: "当天录入信息齐全说明",prop: 'dtlrxxBz',dicData:dictYesNo,span:spanWith},

                { label: "上传数量和录入数量",prop: 'scslylrslxf',dicData:dictYesNo,span:spanWith},
                { label: "上传数量和录入数量说明",prop: 'scslylrslxfBz',dicData:dictYesNo,span:spanWith},
            ]
        },
        {
            label: '检查信息',
            prop: 'check',
            icon: 'el-icon-info',
            column: [
                { label: "Ⅰ检查民警",prop: 'jcmjZh',span:spanWith},
                { label: "Ⅱ检查民警",prop: 'jcmjTwoZh',span:spanWith},
                { label:"检查单位", prop:"jcdwZh",span:spanWith},
                { label:"被检查单位", prop:"bjcdw",span:spanWith},
                { label:"检查时间", prop:"jcsj",
                    format: 'yyyy-MM-dd HH:mm',
                    type: 'datetime',
                    valueFormat: 'yyyy-MM-dd HH:mm',
                    span:spanWith
                },
            ],

        }
    ]
}

File: micro/micro-portal/src/main/java/com/cloud/portal/archive/lvye/model/RoutineCheck.java
package com.cloud.portal.archive.lvye.model;

import com.baomidou.mybatisplus.extension.activerecord.Model;
import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.Date;

/**
 * @author wengshij
 * @date Created in 2019/5/23 13:54
 * @description: 旅馆日常检查信息
 * @modified By:wengshij
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class RoutineCheck extends Model<RoutineCheck> {
    /**
     * 报告制度
     */
    private String bgzd;
    /**
     * 报告制度 备注
     */
    private String bgzdBz;
    /**
     * 被检查单
     */
    private String bjcdw;
    /**
     * 备注
     */
    private String bz;
    /***/
    private String cljg;
    /**
     *
     */
    private String clyj;
    /**
     * 财务报告制度
     */
    private String cwbgzd;
    /**
     * 财务报告制度 备注
     */
    private String cwbgzdBz;
    /**
     * 当天录入信息齐全
     */
    private String dtlrxx;
    /**
     * 当前录入信息备注
     */
    private String dtlrxxBz;
    /**
     * 检查单位
     */
    private String jcdw;
    /**
     * 检查单位中文名称
     */
    private String jcdwZh;
    /**
     * 检查方式
     */
    private String jcfs;
    /**
     * 检查ID
     */
    private String jcId;
    /**
     * 检查民警
     */
    private String jcmj;
    /**
     * 检查民警
     */
    private String jcmjTwo;
    /**
     * 检查民警 -中文
     */
    private String jcmjTwoZh;
    /**
     * 检查民警-中文
     */
    private String jcmjZh;
    /**
     * 检查时间
     */
    private String jcsj;
    /**
     * 来访登记制度
     */
    private String lfdjzd;
    /**
     * 来访登记制度 备注
     */
    private String lfdjzdBz;
    /**
     * 当天录入信息齐全
     */
    private String lkdjylr;
    /**
     * 当前录入信息齐全 备注
     */
    private String lkdjylrBz;
    /**
     * 录入
     */
    private String lr;
    /**
     * 录入人
     */
    private String lrr;
    /**
     * 录入时间
     */
    private String lrsj;
    /***/
    private String ltime;
    /**
     * 旅馆编号
     */
    private String noHotel;
    /**
     * 其他检查人
     */
    private String qtjcr;
    /**
     * 其他检查情况
     */
    private String qtqk;
    /***/
    private String qydzfzr;
    /**
     * 入住登记制度落实 (1合格 0 不合格)
     */
    private String rzdjzd;
    /**
     * 入住登记制度落实备注
     */
    private String rzdjzdBz;
    /**
     * 上传及时
     */
    private String scjs;
    /**
     * 上传及时 备注
     */
    private String scjsBz;
    /**
     * 上传数量与录入数量相符
     */
    private String scslylrslxf;
    /**
     * 传数量与录入数量相符 备注
     */
    private String scslylrslxfBz;
    /**
     * 消防设备
     */
    private String xfsbjhyq;
    /**
     * 消防设备 备注
     */
    private String xfsbjhyqBz;
    /***/
    private String xqzgrq;
    /**
     * 信息系统正常使用
     */
    private String xxxt;
    /**
     * 信息系统正常使用备注
     */
    private String xxxtBz;

    /**
     * 值班巡查制度
     */
    private String zbxczd;
    /**
     * 值班巡查制度 备注
     */
    private String zbxczdBz;
    /**
     * 查询条件-开始时间 字符串
     */
    private String startTime;
    /**
     * 查询条件-结束时间字符串
     */
    private String endTime;


}
